frontend/src/components/globalSearch/GlobalSearchContent.tsx

```diff
@@ -146,17 +146,22 @@ export function GlobalSearchContent({
   // Resource search items
   const resources = useSearchResources();
   const loading = resources.filter(it => it.isLoading).map(it => it.kind);
+  const isMap = useRouteMatch(getClusterPrefixedPath(getDefaultRoutes().map?.path));
+  const location = useLocation();
   const items = useMemo(
     () =>
-      makeKubeObjectResults(resources, item =>
-        history.push(
-          createRouteURL(item.kind, {
-            name: item.metadata.name,
-            namespace: item.metadata.namespace,
-          })
-        )
-      ),
-    [resources]
+      makeKubeObjectResults(resources, item => {
+        const search = new URLSearchParams(location.search);
```

Comment:
Do you need to use the `location` as a dependency for this memo?

Reply:
yep, good catch. (we have to get those react eslint rules configured properly)